### DISCUSSION TITLE
TargetPlatformMinVersion Issue

### DISCUSSION THREAD

**Discussion by MasterThreadInf** [microsoft/microsoft-ui-xaml#8178] [Q&A]
How to resolve Microsoft.UI.Xaml nuget package requires TargetPlatformMinVersion >= 10.0.17763.0 (current project is -1)	issue?

**Comment by robsadams** [comment]
Did you figure this out?

**Reply by DarranRowe** [reply]
Check your .vcxproj/.csproj file.

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net7.0-windows10.0.19041.0</TargetFramework>
    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
    <RootNamespace>CsMeh</RootNamespace>
    <ApplicationManifest>app.manifest</ApplicationManifest>
    <Platforms>x86;x64;ARM64</Platforms>
    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
    <UseWinUI>true</UseWinUI>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.231115000" />
    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.2428" />
    <Manifest Include="$(ApplicationManifest)" />
  </ItemGroup>
</Project>
```
This is a C# (.csproj) project file as an example.

If you have the project set to a Windows Store project type, then this property should be settable via the project properties.

C#
![Screenshot 2024-01-15 104640](https://github.com/microsoft/microsoft-ui-xaml/assets/52577874/fa86e8fc-ebea-4bc8-ade1-e30ebea14b54)

C++
![Screenshot 2024-01-15 104657](https://github.com/microsoft/microsoft-ui-xaml/assets/52577874/2b9bf4fe-de02-4ad0-9e72-344be044f150)

If it isn't, then it doesn't hurt for you to place this element into the global PropertyGroup element. Depending on the project type, you may want to place it into the property group without any labels, like above, or you may want to place it in the group with the Globals label.

```
<?xml version="1.0" encoding="utf-8"?>
<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
<!-- other settings here-->
  <PropertyGroup Label="Globals">
    <VCProjectVersion>16.0</VCProjectVersion>
    <Keyword>Win32Proj</Keyword>
    <ProjectGuid>{a9ed29ff-7a18-4d9f-8132-8a19c301fb9a}</ProjectGuid>
    <RootNamespace>BasicTestWindow</RootNamespace>
  </PropertyGroup>
<!-- other settings here-->
</Project>
```
